### PR TITLE
unity ai speech recognition blog translation completed

### PR DESCRIPTION
**Pull request by SuSung-boy**

### CODE REVIEW

File: zh/unity-asr.md [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+<h1>如何在 Unity 游戏中集成 AI 语音识别？</h1>
+
+<!-- {blog_metadata} -->
+<!-- {authors} -->
+
+[![Open Source AI Game Jam](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/124_ml-for-games/gamejambanner.png)](https://itch.io/jam/open-source-ai-game-jam)
+
+## 简介
+
+语音识别是一项将语音转换为文本的技术，想象一下它如何在游戏中发挥作用？发出命令操纵控制面板或者游戏角色、直接与 NPC 对话、提升交互性等等，都有可能。本文将介绍如何使用 Hugging Face Unity API 在 Unity 游戏中集成 SOTA 语音识别功能。
```
Comment (zhongdongy) [inline zh/unity-asr.md:19]: Why change `state-of-the-art` to its abbreviation?

File: zh/unity-asr.md [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+
+您可以访问 [itch.io 网站](https://individualkex.itch.io/speech-recognition-demo) 下载 Unity 游戏样例，亲自尝试一下语音识别功能。
```
Comment (zhongdongy) [inline zh/unity-asr.md:21]: Missing `Speech Recognition` as attribute.

File: zh/unity-asr.md [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+
+### 先决条件
+
+阅读文本可能需要了解一些 Unity 的基本概念。除此之外，您还需安装 [Hugging Face Unity API](https://github.com/huggingface/unity-api)，可以点击 [之前的博文](https://huggingface.co/blog/unity-api) 阅读 API 安装说明。
```
Comment (zhongdongy) [inline zh/unity-asr.md:25]: During translation of Hugging Face Blog, when you find a link referring another HF blog, please make sure the URL get updated, e.g.:

```plain
https://huggingface.co/blog/unity-api
Should be changed to:
https://huggingface.co/blog/zh/unity-api
```

File: zh/unity-asr.md [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+
+首先我们新建一个 Unity 项目，然后创建一个包含三个 UI 组件的画布 (Canvas)：
+
+1. **开始按钮**: 按下以开始录制语音。
+2. **停止按钮**: 按下以停止录制语音。
+3. **文本组件 (TextMeshPro)**: 显示语音识别结果文本的地方。
+
+### 2. 创建脚本
+
+创建一个名为 `SpeechRecognitionTest` 的脚本，并将其附加到一个空的游戏对象上。
```
Comment (zhongdongy) [inline zh/unity-asr.md:41]: `GameObject` is a unique entity name, I suppose it should be better to keep it or attach after Chinese translation? Like:

```plain
游戏对象 (GameObject)
```

File: zh/unity-asr.md [hunk 5]
```diff
@@ -0,0 +1,369 @@
...
+                    writer.Write((short)(sample * short.MaxValue));
+                }
+            }
+            return memoryStream.ToArray();
+        }
+    }
+}
+```
+
+祝贺！现在您可以在 Unity 游戏中集成 SOTA 语音识别功能了！
```
Comment (zhongdongy) [inline zh/unity-asr.md:367]: Same as a former comment.

**Review by zhongdongy — CHANGES_REQUESTED** [review]
Please also add a new entry in `zh/_blog.yml` file, it should be sorted using `date` field.

File: zh/unity-asr.md (hunk 2 above)
Comment (SuSung-boy) [reply zh/unity-asr.md:21]: 为阅读更顺畅，这里我修改了前后句式，并不影响整体和单句的意思，没太明白 missing 的 语音识别在哪里

File: zh/unity-asr.md (hunk 4 above)
Comment (SuSung-boy) [reply zh/unity-asr.md:41]: OK， I'll modify it

File: zh/unity-asr.md (hunk 3 above)
Comment (SuSung-boy) [reply zh/unity-asr.md:25]: zh page 404, XD

Comment (zhongdongy) [reply zh/unity-asr.md:25]: It's fine. This makes sure we don't need to revise every linkage of a later translated blog. @chenglu What do you think?

**Review by zhongdongy — APPROVED** [review]
Thanks!

**Review by zhongdongy — APPROVED** [review]

**Pull request merged**